pyproject.toml

```diff
@@ -299,3 +279,119 @@ files = [
 [tool.docc.output]
 path = "docs"
 extension = ".html"
+
+[tool.vulture]
+exclude = ["tests/fixtures/", "eest_tests/"]
+ignore_names = ["pytest_*"]
+
+[tool.ruff]
+exclude = [
+    '^\.cache/',
+    '^\.git/',
+    '^\.pytest_cache/',
+    '^\.ruff_cache/',
+    '^\.tox/',
+    '^\.venv/',
+    '^\.vscode/',
+    'tests/fixtures/*',
+    'eest_tests/*',
+    'vulture_whitelist.py',
+]
+line-length = 79
+
+[tool.ruff.lint]
+select = [
+    "E", # pycodestyle errors
+    "F", # Pyflakes
+    "B", # flake8-bugbear
+    "W", # pycodestyle warnings
+    "I", # isort
+    "A", # flake8-builtins
+    "N", # pep8-naming
+    "ARG", # flake8-unused-arguments
+]
+fixable = [
+    "E", # pycodestyle errors
+    "F", # Pyflakes
+    "B", # flake8-bugbear
+    "W", # pycodestyle warnings
+    "I", # isort
+    "D", # pydocstyle
+]
+ignore = [
+# Common to STEEL
+    "D205",     # Missing blank line after summary
+    "D203",     # 1 blank line required before class docstring
+    "D212",     # Multi-line docstring summary should start at the first line
+    "D415",     # First line should end with a ".", "?", or "!"
+
+# Specific to EELS
+    "D107",     # Missing docstring in __init__
+    "D200",     # One-line docstring should fit on one line with quotes
+    "D205",     # 1 blank line required between summary and description
+    "D400",     # First line should end with a period
+    "D401",     # First line should be in imperative mood ("Do", not "Does")
+    "D410",     # Missing blank line after last section (Args, Returns, Raises)
+    "D411",     # Missing blank line before last section
+    "D412",     # No blank lines between sections
+    "D413",     # Missing blank line after last section (same as 410)
+    "D414",     # Section should end with a period
+    "D416",     # Section names should be in the correct order
+    "E203",     # Whitespace before ':'
+]
+
+[tool.ruff.lint.per-file-ignores]
+"tests/*" = [
+    "D100",         # Missing docstring in public module
+    "D101",         # Missing docstring in public class
+    "D103",         # Missing docstring in public function
+    "D104",         # Missing docstring in public package
+    "E501",         # Line too long
+]
+"src/ethereum_spec_tools/evm_tools/loaders/fork_loader.py" = [
+    "N802"          # Property names do not need to be lowercase
+]
+"src/ethereum_spec_tools/lint/*" = [
+    "N802"          # Special linting code absolved of function naming reqs
```

Comment:
Does ruff respect the [`@override`](https://typing.python.org/en/latest/spec/class-compat.html#override) decorator? I _think_ all of those methods are overrides.

Reply:
If you change the method names, they won't get called.